inst/extdata/common/iso_GCAM_regID.csv

```diff
@@ -1,244 +1,244 @@
-# File:iso_GCAM_regID.csv,,,
```

Comment:
Not entirely sure what the deal is with this. I  don't think I replaced this file... maybe opened it in excel and induced new commas or something.